index.html

```diff
@@ -13,7 +13,7 @@ <h1>Dev Delta Jokes</h1>
     <main>
       <ul>
         <li><a href="mikel/index.html">Mikel's joke page</a></li>
-        <li>Collin Placeholder</li>
+        <li><a href="collin/index.html">Collin's Joke</a>></li>
```

Comment:
Is the right bracket here intentional? 

![image](https://user-images.githubusercontent.com/13723156/102023026-4f2d9f00-3d50-11eb-8c51-399f4f92bf24.png)